index.html

```diff
@@ -121,6 +130,115 @@ <h2 class="txt-center">
     $('.js-defaultServerIndexApp').on('click', function(){
       app.loadNewServer("http://localhost:18469/api/v1/");
     });
+
+    var appMenu = Menu.buildFromTemplate([
```

Comment:
I meant main.js, not js/main.js. That's where the other menus are. I guess it's not that huge of a deal.